components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -47,71 +61,11 @@ const getChartOptions = (intl, startDate, endDate, hostCreatedAt) => {
   };
 };
 
-const getCategories = (intl, startDate, endDate, hostCreatedAt) => {
-  const numberOfDays = dayjs(startDate || hostCreatedAt).diff(dayjs(endDate), 'day');
-  if (numberOfDays <= 7) {
-    const startDay = startDate.getDay();
-    return [...new Array(7)].map((_, idx) =>
-      intl.formatDate(new Date(0, 0, idx + startDay), { weekday: 'long' }).toUpperCase(),
-    );
-  } else if (numberOfDays <= 365) {
-    const currentMonth = new Date().getMonth();
-    return [...new Array(12)].map((_, idx) =>
-      intl.formatDate(new Date(0, idx + currentMonth + 1), { month: 'short' }).toUpperCase(),
-    );
-  } else {
-    return [...new Array(6)].map((_, idx) =>
-      intl.formatDate(new Date(new Date().getFullYear() - 5 + idx, 0), { year: 'numeric' }).toUpperCase(),
-    );
-  }
-};
-
-const getCategoryType = (startDate, endDate, hostCreatedAt) => {
-  const numberOfDays = dayjs(startDate || hostCreatedAt).diff(dayjs(endDate), 'day');
-  if (numberOfDays <= 7) {
-    return 'WEEK';
-  } else if (numberOfDays <= 365) {
-    return 'MONTH';
-  } else {
-    return 'YEAR';
-  }
-};
-
 const constructChartDataPoints = (category, dataPoints) => {
```

Comment:
I'm surprised that ESLint doesn't complain with `category` not being used?

Reply:
Yes, I was surprised too, but somehow it doesn't complain. Anyways I've remove the `category` arg. 🤔